pkgs/development/python-modules/mockito/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, buildPythonPackage, fetchPypi, isPy3k, funcsigs, pytest, numpy }:
+
+buildPythonPackage rec {
+  version = "1.1.1";
+  pname = "mockito";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "142f5e8865a422ad2d67b9c67a382e3296e8f1633dbccd0e322180fba7d5303d";
+  };
+
+  # Failing tests due 2to3
+  doCheck = !isPy3k;
+
+  propagatedBuildInputs = stdenv.lib.optionals (!isPy3k) [ funcsigs ];
+  checkInputs = [ pytest numpy ];
+
+  checkPhase = ''
+    pytest
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Spying framework";
```

Comment:
While the [README of the mockito-python](https://github.com/kaste/mockito-python/blob/1.1.1/README.rst) indeed calls it a "spying framework", I think that label somewhat confusing out of context. It's only for "spying" on the system-under-test, specifically how that uses the provided mocks, partial mocks and stubs. It's not for spying on, say, people or organizations.

Thus I'd go with how the [Java library](https://github.com/mockito/mockito) [of the same name](https://site.mockito.org/) calls itself: A "mocking framework".

Reply:
I'm personally inclined to use the description used by the project itself, but I can change it if you prefer it